### PR TITLE
[Haskell] Add maintainers and link to Haskell-specific v3 GitHub issues

### PR DESCRIPTION
**Pull request by sshine**

### CODE REVIEW

File: languages/haskell/maintainers.md [hunk 1]
```diff
@@ -4,8 +4,10 @@ These awesome people help maintain the Haskell track.
 
 ## Senior Maintainers
 
-TODO: add senior maintainers
+[@petertseng](https://github.com/petertseng)
```
Comment (tehsphinx) [inline languages/haskell/maintainers.md]: This was probably done before the pattern emerged (see https://github.com/exercism/v3/blob/master/languages/csharp/maintainers.md)

File: README.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ We are busy working away on the product and development work to add all this fun
 - [C#](https://github.com/exercism/v3/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Astatus%2Fhelp-wanted+label%3Atrack%2Fcsharp)
 - [Go](https://github.com/exercism/v3/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Astatus%2Fhelp-wanted+label%3Atrack%2Fgo)
 - [F#](https://github.com/exercism/v3/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Astatus%2Fhelp-wanted+label%3Atrack%2Ffsharp)
+- [Haskell](https://github.com/exercism/v3/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Astatus%2Fhelp-wanted+label%3Atrack%2Fhaskell)
```
Comment (tehsphinx) [inline README.md]: This needs a rebase...

File: languages/haskell/maintainers.md (hunk 1 above)
Comment (sshine) [reply languages/haskell/maintainers.md]: Where is that pattern documented?

Comment (sshine) [reply languages/haskell/maintainers.md]: Never mind, I've adhered to it. :)

File: README.md [hunk 3]
```diff
@@ -18,6 +18,7 @@ We are busy working away on the product and development work to add all this fun
 - [C#](https://github.com/exercism/v3/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Astatus%2Fhelp-wanted+label%3Atrack%2Fcsharp)
 - [Elixir](https://github.com/exercism/v3/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Astatus%2Fhelp-wanted+label%3Atrack%2Felixir)
 - [F#](https://github.com/exercism/v3/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Astatus%2Fhelp-wanted+label%3Atrack%2Ffsharp)
+- [Haskell](https://github.com/exercism/v3/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Astatus%2Fhelp-wanted+label%3Atrack%2Fhaskell)
 - [Go](https://github.com/exercism/v3/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Astatus%2Fhelp-wanted+label%3Atrack%2Fgo)
```
Comment (tehsphinx) [inline README.md]: Didn't see this before... can you sort this alphabetically?

Comment (sshine) [reply README.md]: Why not!

File: languages/haskell/README.md [hunk 4]
```diff
@@ -13,8 +13,8 @@ This area will contain everything needed to launch the Haskell track, including:
 Before we publicize requesting contribution for this language, the following steps should be done.
 
 - [ ] Have a kick-off discussion between track maintainers
-- [ ] Fill out the [maintainers.md](./maintainers.md) file (e.g. [C#](../csharp/maintainers.md))
-- [ ] Ensure there is a link to your track's GitHub issues on the [main README.md](../../README.md)
+- [x] Fill out the [maintainers.md](./maintainers.md) file (e.g. [C#](../csharp/maintainers.md))
```
Comment (petertseng) [inline languages/haskell/README.md]: C#?

Comment (petertseng) [reply languages/haskell/README.md]: oh that's an example. all good.

**Pull request merged**